src/dust_rng.cpp

```diff
@@ -165,3 +165,18 @@ cpp11::writable::raws dust_rng_state(SEXP ptr) {
   std::memcpy(RAW(ret), state.data(), len);
   return ret;
 }
+
+// This is is not for general use, trying to trigger a bug
+[[cpp11::register]]
+cpp11::sexp test_rbinom_float(cpp11::sexp r_seed, int n_samples,
+                              int size, double p) {
```

Comment:
Are we keeping this in this PR?

Reply:
until the coverage starts having a complain, yeah. This triggers the bad case quite quickly and will be the starting point for finding what the correct eps is